content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -162,6 +159,12 @@ The output shows that the PersistentVolumeClaim is bound to your PersistentVolum
     NAME            STATUS    VOLUME           CAPACITY   ACCESSMODES   STORAGECLASS   AGE
     task-pv-claim   Bound     task-pv-volume   10Gi       RWO           manual         30s
 
+Running pod with the duplicate persistent volume.
+
+{{< codenew file="pods/storage/pv-duplicate.yaml" >}}
+
+Example : A container running a web application reads application data from a persistent volume and saves logs to another persistent volume.
```

Comment:
This example doesn't illustrate what you're achieving with the yaml.

Reply:
can you suggest what should be the example?

Reply:
you could do 2 volume mounts for your nginx container:
- /usr/share/nginx/html for the static website
- /etc/nginx/nginx.conf for the default config

Reply:
obviously you need to change the yaml example to match the text ;-)